dags/create_newsletter.py

```diff
@@ -14,11 +13,9 @@
 )
 
 
-logger = logging.getLogger(__name__)
-
 
 @asset(schedule="@daily")
-def raw_zen_quotes() -> list[dict]:
+def raw_zen_quotes(context: dict):
```

Comment:
The generator created by yield does not like the typing here. It does not error but the IDE flags it as incorrect, thats why I removed it since it is not necessary.